cmd/pebble/cmd_run.go

```diff
@@ -52,6 +53,7 @@ func init() {
 			"hold":        "Do not start default services automatically",
 			"http":        `Start HTTP API listening on this address (e.g., ":4000")`,
 			"verbose":     "Log all output from services to stdout",
+			"args":        `Provide arguments to service (e.g., "SERVICE ARGS.. ; ")`,
```

Comment:
That is a better plan, I agree. 

I am just wondering if we can provide arguments to **startup-enabled** services only. :thinking:

Reply:
Hi Ben, I removed the `startup-enabled` part, because right now, arguments can be passed even if they are not startup-enabled. Let me know if you think it should only be passed for startup-enabled services.